runtime/complex.go

```diff
@@ -68,5 +69,80 @@ func complexRepr(f *Frame, o *Object) (*Object, *BaseException) {
 }
 
 func initComplexType(dict map[string]*Object) {
+	ComplexType.slots.Eq = &binaryOpSlot{complexEq}
+	ComplexType.slots.GE = &binaryOpSlot{complexGE}
+	ComplexType.slots.GT = &binaryOpSlot{complexGT}
+	ComplexType.slots.LE = &binaryOpSlot{complexLE}
+	ComplexType.slots.LT = &binaryOpSlot{complexLT}
+	ComplexType.slots.NE = &binaryOpSlot{complexNE}
 	ComplexType.slots.Repr = &unaryOpSlot{complexRepr}
 }
+
+func complexEq(f *Frame, v, w *Object) (*Object, *BaseException) {
+	e, ok := complexCompare(toComplexUnsafe(v), w)
+	if !ok {
+		return NotImplemented, nil
+	}
+	return GetBool(e).ToObject(), nil
+}
+
+func complexGE(f *Frame, v, w *Object) (*Object, *BaseException) {
```

Comment:
Rather than writing a wrapper function for each of complexLE, GE, etc. I would just make comparisonNotSupported compatible with binary op and then use it for all the unsupported slots:

```
ComplexType.slots.GE = &binaryOpSlot{comparisonNotSupported}
ComplexType.slots.GT = &binaryOpSlot{comparisonNotSupported}
...
```